### ISSUE TITLE
Sometimes dash corners are not enough

### ISSUE THREAD

**Issue by samlaminilepre** [OpenOrienteering/mapper#1506]
### Steps to reproduce
1.Draw a cliff with 90° angles
If you use a dash corners is even worst

### Actual behaviour
Tag lines will cross each other, if you use a dash corners is even worst

### Expected behaviour
It would be great to be able to delete a mid simbol when needed, without having to cut it and use another simbol without tag lines or other mid simbol.
### Configuration
Mapper Version:0.9.1
Operating System:windows
![scarpate](https://user-images.githubusercontent.com/45790530/73120557-03270f80-3f70-11ea-9a33-13c3484f25c6.JPG)

**Comment by rem-maps** [comment]
I also have problems with dash corners - can someone help?
Maybe there is already a way to solve my problem but I can't find it.

Readability is a major target for me - I want my maps to be 'clean' and readable.

In the attached pictured you see what I want to achieve - but I cannot achieve it either with normal corner point (I don't want the mid point tag so near the fence corner) or with dash point (I don't want the mid point tag in that short section of fence) .

The only workaround I can figure out is to duplicate symbol and have one of them without flag 'show at least one mid symbol'.
But I should duplicate too many symbols and cut fence into two or more parts and I don't like it.

Basically I would like to have a type of point that removes any mid point nearby.

Thanks

![fence](https://user-images.githubusercontent.com/59602742/73557092-e77ca700-4450-11ea-9e4f-c75400c155d4.png)

**Comment by mlerjen** [comment]
@rem-maps 
![image](https://user-images.githubusercontent.com/15611653/80283413-a296a280-8717-11ea-936d-7c785525b45a.png)
By deactivating the "Show at least one mid symbol" option you can shoo away tags from certain sections.
![image](https://user-images.githubusercontent.com/15611653/80283501-4c762f00-8718-11ea-96c7-4ddace8c0384.png)

**Comment by rem-maps** [comment]
@mlerjen  I don't get your sentence at all -  I don't understand if there are spelling mistakes or just I don't understnad your english. Can you re-write it?
>>Modified symbol opted out the SALOMS-option but also with dash. see just below here.
By deactivating the "Show at least one mid symbol" option you can shoo away tags from certain sections.

is your suggestion different from what I already said?
>>The only workaround I can figure out is to duplicate symbol and have one of them without flag 'show at least one mid symbol'.

**Comment by mlerjen** [comment]
@rem-maps: my bad.

**Comment by rem-maps** [comment]
I noticed that 
in ISSPROM default symbol set  
"Show at least one mid symbol" is selected for fence  (516) and not selected for impassable fence (518).

In ISOM default symbol set  
"Show at least one mid symbol" is selected for both fence  (516) and impassable fence (518).

Is there any reason?

My opinion is that I would like them both to have it NOT selected in the default symbol set (both ISSPROM and ISOM)

version 0,9.2

**Comment by mlerjen** [comment]
While to opt out the "Show at least one mid symbol" works for the fences, it seems not to solve the original problem of @samlaminilepre . Cliffs and earth banks seem to behave according to the option even while the later is deselected.
![image](https://user-images.githubusercontent.com/15611653/80461841-2e861580-8936-11ea-8df4-64f0df79592d.png)
 If the distance between two dash points is shorter than the "distance from line end" no mid-symbol should be shown... or should it?

**Comment by dg0yt** [comment]
@mlerjen I don't know an easy solution, except cutting and mixing top line symbol and symbol with tag lines.

![screenshot](https://user-images.githubusercontent.com/13567791/80675087-e7259380-8ab3-11ea-922c-fae8e194d5dd.png)


Whatever we implement as complex behaviour of a single symbol, it will be hard/impossible to make this interoperable with other software such as OCAD or Purple Pen. PDF will work, of course.

**Comment by eolmapper** [comment]
I think the problem is for OOM like Ocad, the dash point is considered like the end of the line. It uses the "end of line distance" parameter for OOM or "b" for Ocad to separate the tag and the dash point.
For some symbols, this distance is very small and the dash point does not work as wanted.
I think it should be the "distance between spots" or ("a" in Ocad) that should be used instead. A dash point separates the spots by 2x the distance between the spots.

For now, I have solved this problem for somes symbols which always have at least 2 spots.
I add "start symbol" and "end symbol", and I increase the "distance from the end of the line" for the midpoint by the value of "distance between two points".
